docs/test_instructions_per_dependency_update.md

```diff
@@ -421,6 +422,19 @@ Step.3:
 
 -----
 
+### PlayReview [[googlePlayReviewVersion](https://github.com/wordpress-mobile/WordPress-Android/blob/trunk/build.gradle)] <a name="playreview"></a>
+
+<details>
+    <summary>1. In app reviews</summary>
+
+- Perform a clean install.
+- Publish three (`ReviewViewModel.TARGET_COUNT_POST_PUBLISHED`) new posts or stories.
```

Comment:
Is this supposed to be `Publish two`?

Reply:
I intentionally typed three to confirm that it does not crash after `TARGET_COUNT_POST_PUBLISHED` is reached.  I updated the parentheses information from `ReviewViewModel.TARGET_COUNT_POST_PUBLISHED` to `ReviewViewModel.TARGET_COUNT_POST_PUBLISHED + 1`: 240ed435cf1c264a073f954a02706a1c085dfc0f